iree/compiler/Conversion/LinalgToSPIRV/ConvertToGPUPass.cpp

```diff
@@ -659,6 +644,28 @@ LogicalResult mapLinalgOpToLocalInvocationIdImpl<linalg::CopyOp>(
   return distributeCyclicallyToProcessors(rewriter, pLoopOp, idAndCount.id,
                                           idAndCount.count);
 }
+/// CopyOp that are loading to/storing from workgroup memory are special cased
```

Comment:
Add a blank line before the comment can help readability.

https://google.github.io/styleguide/cppguide.html#Vertical_Whitespace

Reply:
Done.